<launch>
  <arg name="record_path" doc="folder where the bag file will be stored" />
  <node name="simulation_recorder" pkg="rosbag"
    type="record" args="-e '/gazebo/model_states|(.*)scan|(.*)joint_states' 
      -o $(arg record_path)" />
  
</launch>